### PR TITLE
main/subversion: upgrade to 1.12.0

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/subversion/APKBUILD [hunk 1]
```diff
@@ -64,12 +64,10 @@ build() {
 }
 
 check() {
-	cd "$builddir"
```
Comment (tcely) [inline main/subversion/APKBUILD]: This package is in `main` so we keep these `cd` lines to make back porting easier.

File: main/subversion/APKBUILD [hunk 2]
```diff
@@ -64,12 +64,10 @@ build() {
...
 	make test
 }
 
 package() {
-	cd "$builddir"
```
Comment (tcely) [inline main/subversion/APKBUILD]: Keep this `cd` line.

File: main/subversion/APKBUILD (hunk 1 above)
Comment (J0WI) [reply main/subversion/APKBUILD]: Done

**Comment by J0WI** [comment]
dbc32a6c7771ef26009ddd2151d9e67c0ee3ab01

**Pull request closed without merging**